modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -111,7 +114,7 @@ When `linkType` is set to `ib`, `isRdma` is automatically set to `true` by the S
 +
 Do not set linkType to `eth` for SriovNetworkNodePolicy, because this can lead to an incorrect number of available devices reported by the device plugin.
 
-<19> Optional: To enable hardware offloading, the `eSwitchMode` field must be set to `"switchdev"`.
+<19> Optional: To enable hardware offloading, the `eSwitchMode` field must be set to `"switchdev"`. For more information about hardware offloading , see Configuring hardware offloading in the Additional resources section.
```

Comment:
```suggestion
<19> Optional: To enable hardware offloading, you must set the `eSwitchMode` field to `"switchdev"`. For more information about hardware offloading , see "Configuring hardware offloading". 
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

> Do not state "in the Additional resources section" or other variants.